### PR TITLE
fix on the progress bar feature of the management dashboard

### PR DESCRIPTION
**Pull request by IamDavidOkeke**

### CODE REVIEW

**Comment by esk47** [comment]
The logic for changing the color of the progress bar isn't there.

**Comment by EvianTan** [comment]
Actually I'm not sure when should we include the changed `package-lock.json` file in PR, could you help me answer this? @esk47 On my side, the `package-lock.json` file always changed when I made any change in code, but when I pushed the code to remote I always ignored this kind of json files...

**Comment by EvianTan** [comment]
Actually I'm not sure when should we include the changed `package-lock.json` file in PR, could you help me answer this? @esk47 On my side, the `package-lock.json` file always changed when I made any change in code, but when I pushed the code to remote I always ignored this kind of json files...

**Comment by esk47** [comment]
Yes, package-lock.json should be included. 

I was not 100% so I checked too:
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

File: src/components/TeamMemberTasks/TeamMemberTasks.jsx [hunk 1]
```diff
@@ -226,10 +230,31 @@ const TeamMemberTasks = props => {
...
+//Add time entry property to task
+                let allTimeEntries = []
+                finalData.forEach((member)=>{
+                  member.tasks.forEach((task)=>{
+                  task.allTimeEntries = allTimeEntries
+                  member.timeEntries.forEach((entry)=>{
+                    if (task.projectId === entry.projectId ){
+                   task.allTimeEntries.push(entry)
+                  
+                    }else{
```
Comment (EvianTan) [inline src/components/TeamMemberTasks/TeamMemberTasks.jsx:243]: The code format here looks a bit weird, did you have Prettier installed @IamDavidOkeke 
image

Comment (EvianTan) [inline src/components/TeamMemberTasks/TeamMemberTasks.jsx:243]: The code format here looks a bit weird, did you have Prettier installed @IamDavidOkeke 
image

File: src/components/TeamMemberTasks/TeamMemberTasks.jsx [hunk 2]
```diff
@@ -304,7 +329,27 @@ const TeamMemberTasks = props => {
               </p>
             ))}
         </td>
-        <td>tempprogress</td>
+        <td  className=''>
+          {
```
Comment (EvianTan) [inline src/components/TeamMemberTasks/TeamMemberTasks.jsx:333]: Same thing as the comment near line 243, please also re-format this part.

**Comment by EvianTan** [comment]
Now the task and it's progress number and progress bar are not aligned. I'm thinking about you can split them as the progress part in leaderboard below. Still need @esk47 's advice here 
image
image

**Comment by EvianTan** [comment]
When the functional code changes are done, feel free to remove your debug output like "console.log()" and unmeaningful comments 😃

File: src/components/TeamMemberTasks/TeamMemberTasks.jsx [hunk 3]
```diff
@@ -304,7 +329,27 @@ const TeamMemberTasks = props => {
               </p>
             ))}
         </td>
-        <td>tempprogress</td>
+        <td  className=''>
+          { 
+             member.tasks &&
+             member.tasks.map((task, index)=>{ 
+               const value = task.allTimeEntries? Math.floor(task.allTimeEntries.reduce(
```
Comment (EvianTan) [inline src/components/TeamMemberTasks/TeamMemberTasks.jsx:336]: Consider use `round()` instead of `floor` here. Same thing as line 341

**Comment by EvianTan** [comment]
This PR didn't pass the build check, please fix it.
image
I took a brief look on that, that might because you repeated line 157-192, please fix them.

**Pull request closed without merging**